virt/virt-4-10-release-notes.adoc

```diff
@@ -68,6 +68,7 @@ The SVVP Certification applies to:
 === Networking
 
 //CNV-11201 Rollout of NodeNetworkConfigurationPolicy
+* You can now xref:../virt/node_network/virt-updating-node-network-config.adoc#virt-creating-interface-on-nodes_virt-updating-node-network-config[configure multiple nmstate-enabled nodes concurrently] by using a single `NodeNetworkConfigurationPolicy` manifest.
 
```

Comment:
Why does the xref take to the `Creating an interface on nodes` section?
Based on the sentence given here, seems to me it should lead to the `Updating node network configuration
`, which is the actually the header of the entire page.
WDYT?

Reply:
Sorry, I didn't read the entire paragraph.
LGTM